index.js

```diff
@@ -9,11 +18,86 @@ wrapId.addEventListener('click', () => {
   rightNavebarId.classList.toggle('change-rightnavbar');
 });
 
+const popupArray = [{
+  Name: 'Keeping track of hundreds  of components website',
+  Name1: 'Multi Post Stories',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  featureimage: './images/popupimage.png',
+  technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+  linktoliveversion: '#',
+  linktosource: '#',
+}];
+
+for (let i = 0; i < seeprojectbutton4.length; i++) {
+  seeprojectbutton4[i].addEventListener('click', () => {
+    portfolio.classList.toggle('active');
+    navbar.classList.toggle('active');
+    headlinesection.classList.toggle('active');
+    aboutme.classList.toggle('active');
+    contactsection.classList.toggle('active');
+    footercontainer.classList.toggle('active');
+    // eslint-disable-next-line no-unused-vars
+    popupArray.forEach((items, index) => {
+      const container = document.querySelector('.container');
+      const section = document.createElement('section');
+      container.appendChild(section);
+      section.innerHTML = `<div class="pop-up-container">
+      <header class="pop-up-header">
+        <div class="pop-up-heading">
+          <h2 id="pop-up-heading">${items.Name}</h2>
+          <button class="close-btn">&times;</button> 
+        </div>
+        <div class="pop-up-buttons">
+          <span class="pop-up-lang-btn">${items.technologies[0]}</span>
+          <span class="pop-up-lang-btn">${items.technologies[1]}</span>
+          <span class="pop-up-lang-btn">${items.technologies[2]}</span>
+        </div>
+      </header>
+      <section class="pop-up-body">
+        <div class="pop-up-image">
+          <img src="${items.featureimage}" alt="Projectimg">
+        </div>
+        <div class="pop-up-details">
+          <p>${items.description}</p>
+          <div class="pop-up-details-btn-container">
+            <a href="${items.linktoliveversion}"><button class="pop-up-details-btn">See Live <span><img src="./images/icon-see-live.png" class='icon-see-live-img' alt="iconseeimage"></span></button></a>
```

Comment:
- [ ] As requeted by the previous reviewer, please make sure your image on the `see live` button is not broken.

   - ### YOUR DESIGN
   
      ![image](https://user-images.githubusercontent.com/87197412/216327910-bff86ccf-184f-43c4-8c83-dc496e1e0ecd.png)

Reply:
![image](https://user-images.githubusercontent.com/116083031/216334616-4c02d882-1db1-4844-915a-daf197eaadbf.png)
Please kindly see its working for me.